### PR TITLE
Fix the issue of submenus not appearing

### PR DESCRIPTION
**Pull request by kienstra**
#### Changes
* Ensures that on first activation, the block CPT menus appear:
the-submenu-items-appear

* This bug only appeared the first time GCB is *ever* activated on the site. 
#### Testing instructions
* Please see the steps to reproduce in BHH-554
* To destroy the site and create a new one, I've been doing:
```
wp db drop && wp db create && wp core install --url=https://foo.test --title="Example" --admin_user=foo --admin_email=example@example.com --admin_password=baz
```

### CODE REVIEW

File: php/PostTypes/BlockPost.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function __construct() {
 	 */
 	public function register_hooks() {
 		add_action( 'init', [ $this, 'register_post_type' ] );
-		add_action( 'admin_init', [ $this, 'add_caps' ] );
+		add_action( 'plugins_loaded', [ $this, 'add_caps' ] );
```
Comment (kienstra) [inline php/PostTypes/BlockPost.php:50]: Maybe `admin_init` was a little too late to add the capabilities. 

By moving it earlier to `plugins_loaded`, the capability is there in time for the submenus to be there.

**Comment by kienstra** [comment]
It should also be possible to revert https://github.com/studiopress/genesis-custom-blocks/pull/23

**Comment by kienstra** [comment]
Hi @mindctrl and @nickcernis,
Could you please review this if you have a chance? Not urgent 😄

**Review by nickcernis — APPROVED** [review]
👍 Confirmed “Add New” and “All Blocks” appear on first use on a fresh install.

Screenshot 2020-10-19 at 17 32 02

**Review by mindctrl — APPROVED** [review]

**Comment by kienstra** [comment]
Nice, thanks so much for reviewing this!

**Pull request merged**